src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,17 +30,26 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'position_after_functions_and_oop_constructs' => self::NEXT_LINE,
```

Comment:
I suggested to use a boolean option because your current implementation seems to allow two forced positions for the braces but not to leave them untouched. But if you actually want to support disabling this as well, then a string option makes sense, altought I think another fixer has a similar option and uses `true|false|null`.